rskj-core/src/main/java/org/ethereum/vm/VM.java

```diff
@@ -1393,6 +1406,9 @@ protected void doCALL(){
         // value is always zero in a DELEGATECALL operation
         DataWord value = op.equals(OpCode.DELEGATECALL) ? DataWord.ZERO : program.stackPop();
 
+        if (program.isStaticCall() && op == CALL && !value.isZero())
```

Comment:
You have missing curly braces in this block

Reply:
Fixed